test/jdk/java/net/httpclient/whitebox/java.net.http/jdk/internal/net/http/SimpleSSLContext.java

```diff
@@ -42,13 +42,6 @@
 /**
  * Creates a simple usable SSLContext for SSLSocketFactory
  * or a HttpsServer using a default keystore in the test tree.
- * <p>
- * Using this class with a security manager requires the following
- * permissions to be granted:
- * <p>
- * permission "java.util.PropertyPermission" "test.src.path", "read";
- * permission java.io.FilePermission "/path/to/test/lib/jdk/test/lib/testkeys", "read";
- * The exact path above depends on the location of the test.
  */
```

Comment:
Can the securityExceptions local variable be removed? And the catch (SecurityException) {} block?

Reply:
hmmm... probably. if there's not any custom code that might throw it in there, we can remove it. good point.

Reply:
Done